data/mock/addressConverterMock.go

```diff
@@ -0,0 +1,66 @@
+package mock
+
+import (
+	"bytes"
+	"encoding/hex"
+	"errors"
+
+	"github.com/ElrondNetwork/elrond-go/data/state"
+)
+
+var errFailure = errors.New("failure")
+
+type AddressConverterMock struct {
+	Fail                                          bool
+	CreateAddressFromPublicKeyBytesRetErrForValue []byte
+}
+
+func (acm *AddressConverterMock) CreateAddressFromPublicKeyBytes(pubKey []byte) (state.AddressContainer, error) {
+	if acm.Fail {
+		return nil, errFailure
+	}
+
+	if acm.CreateAddressFromPublicKeyBytesRetErrForValue != nil {
+		if bytes.Equal(acm.CreateAddressFromPublicKeyBytesRetErrForValue, pubKey) {
+			return nil, errors.New("error required")
+		}
+	}
+
+	return NewAddressMockFromBytes(pubKey), nil
+}
+
+func (acm *AddressConverterMock) ConvertToHex(addressContainer state.AddressContainer) (string, error) {
+	if acm.Fail {
+		return "", errFailure
+	}
+
+	return hex.EncodeToString(addressContainer.Bytes()), nil
+}
+
+func (acm *AddressConverterMock) CreateAddressFromHex(hexAddress string) (state.AddressContainer, error) {
+	if acm.Fail {
+		return nil, errFailure
+	}
+
```

Comment:
implement mock functions, panic is not preferred. use a mock default implementation, if the "Called" function is not implemented.

Reply:
This was mock copied over from process.